packages/blocks/src/page-block/edgeless/utils/panning-utils.ts

```diff
@@ -0,0 +1,47 @@
+import type { PointerEventState } from '@blocksuite/block-std';
+
+import type { Renderer } from '../../../surface-block/renderer.js';
+
+export type MoveDelta = {
```

Comment:
Can this be named as a generic type, something like IVec?
Or if interested, could we standardize the vector type to IVec?